tests/foreman/api/test_product.py

```diff
@@ -295,3 +300,51 @@ def test_positive_delete(self):
                 product.delete()
                 with self.assertRaises(HTTPError):
                     entities.Product(id=product.id).read()
+
+    @tier1
+    def test_positive_sync(self):
+        """Sync repository for a product
```

Comment:
However documentation for that API path says 'Sync all repositories for a product' :)